services/metadactyl-clj/src/metadactyl/metadata/reference_genomes.clj

```diff
@@ -111,8 +109,8 @@
   "Validates a single field in a reference genome."
   [genome field]
   (when (blank? (genome field))
-    (throw+ {:error_code error-codes/ERR_BAD_OR_MISSING_FIELD
-             field "empty required field"
+    (throw+ {:type       :clojure-commons.exception/missing-request-field
+             :error      "empty required field"
```

Comment:
This makes it so the new error doesn't show which field is missing, where the old one did. Maybe `(str "empty required field: " field)` or so?